types/react-native/index.d.ts

```diff
@@ -3491,7 +3491,7 @@ export interface FlatListProperties<ItemT> extends VirtualizedListProperties<Ite
     /**
      * Rendered in between each item, but not at the top or bottom
      */
-    ItemSeparatorComponent?: React.ComponentClass<any> | null
+    ItemSeparatorComponent?: React.ComponentClass<any> | React.ReactElement<any> | (() => React.ReactElement<any>) | null
```

Comment:
I’m confused, [we determined](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/18823#issuecomment-321985718) that this was supposed to be just: `React.ComponentClass<any> | null`, right?

Afaik it should definitely _not_ have `React.ReactElement<any>`.